src/Lifecycle.ts

```diff
@@ -179,6 +183,12 @@ export function attemptTokenLogin(
     const identityServer = localStorage.getItem(SSO_ID_SERVER_URL_KEY);
     if (!homeserver) {
         console.warn("Cannot log in with token: can't determine HS URL to use");
+        Modal.createTrackedDialog("SSO", "Unknown HS", ErrorDialog, {
+            title: _t("We couldn't log you in"),
+            description: _t("We asked the browser to remember which homeserver you use, to let you sign in. " +
+                "Unfortunately your browser has forgotten in. Go to the sign in page and try again."),
```

Comment:
Could perhaps use some Design copy editing, but here's my minimal attempt:

```suggestion
            description: _t("We asked the browser to remember which homeserver you use to let you sign in, " +
                "but unfortunately your browser has forgotten it. Go to the sign in page and try again."),
```

Reply:
The copy was written by design but this seems easier to read